crates/algorithms/randomness/src/bin/passwd.rs

```diff
@@ -0,0 +1,13 @@
+use rand::Rng;
+use rand_distr::Alphanumeric;
+
+fn main() {
+    const PASSWORD_LEN: usize = 30;
+    let mut rng = rand::rng();
+
+    let password: String = (0..PASSWORD_LEN)
+        .map(|_| rng.sample(Alphanumeric) as char)
+        .collect();
```

Comment:
The example from where this was copied was not using Alphanumeric, but the text said it was. Fixed this in this change.